docs/ref/contrib/auth.txt

```diff
@@ -315,6 +315,14 @@ Manager methods
         Same as :meth:`create_user`, but sets :attr:`~models.User.is_staff` and
         :attr:`~models.User.is_superuser` to ``True``.
 
+    .. method:: with_perm(perm, obj=None)
+
+        .. versionadded:: 1.11
+
+        Returns a queryset containing :class:`~django.contrib.auth.models.User`
+        objects that have the given permission ``perm`` in the form of
+        ``"<app label>.<permission codename>"``, including superusers.
```

Comment:
I'm not sure about the use cases. For example, someone might want to get users with permissions for all backends. It would be nice if we had some feedback about what users are implementing on their own to confirm we're targeting the largest use case.